build/AzurePipelinesTemplates/WindowsAppSDK-PackTransportPackage-Stage.yml

```diff
@@ -193,33 +193,12 @@ stages:
     #       arguments: -Platform "x64" -Configuration "release" -AzureBuildStep "BuildMock" -OutputDirectory "$(build.artifactStagingDirectory)\FullNuget" -PackageVersion "$(packageVersion)"
 
     - ${{ if eq(parameters.SignOutput, 'true') }}:
-      - task: EsrpCodeSigning@2
-        inputs:
-          ConnectedServiceName: 'Project Reunion ESRP Code Signing Connection'
+      - template: AzurePipelinesTemplates/WindowsAppSDK-EsrpCodeSigning-Steps.yml@WindowsAppSDKConfig
+        parameters:
           FolderPath: $(build.artifactStagingDirectory)\FullNuget
           Pattern: 'Microsoft.WindowsAppSDK.Foundation*.nupkg'
+          KeyCode: 'CP-401405'
```

Comment:
Nonblocking - In other files, KeyCode is followed by displayName, is it intentional to drop displayName here? I know this won't affect functionality.

Reply:
Yea I will add a display name here. Not intentional